doc/source/whatsnew/v0.18.2.txt

```diff
@@ -19,10 +19,66 @@ Highlights include:
 New features
 ~~~~~~~~~~~~
 
+.. _whatsnew_0182.enhancements.read_csv_dupe_col_names:
 
+Duplicate column names are now supported in ``pd.read_csv()`` whether they are in the file
+or passed in as the `names` parameter (:issue:`7160`, :issue:`9424`)
 
+.. ipython :: python
 
+   data = '0,1,2\n3,4,5'
+   names = ['a', 'b', 'a']
 
+Previous behaviour:
+
+.. code-block:: ipython
+
+   In [2]: pd.read_csv(StringIO(data), names=names)
+   Out[2]:
+      a  b  a
+   0  2  1  2
+   1  5  4  5
+
+The first 'a' column contains the same data as the second 'a' column, when it should have
+contained the array ``[0, 3]``.
+
+New behaviour:
+
+.. code-block:: ipython
+
+   In [2]: pd.read_csv(StringIO(data), names=names)
+   Out[2]:
+      a  b  a.1
+   0  0  1    2
+   1  3  4    5
+
+The same result above can be obtained if the duplicate column names were in the header of the
+file itself:
+
+.. code-block:: ipython
+
+   In [2]: data = 'a,b,a\n0,1,2\n3,4,5'
+   In [3]: pd.read_csv(StringIO(data))
+   Out[3]:
+      a  b  a.1
+   0  0  1    2
+   1  3  4    5
+
+There is no more duplicate data because by default, ``mangle_dupe_cols`` is set to ``True``,
```

Comment:
``mangle_dupe_cols=True`` (say this parameter setting is unchanged from preivous versions)